pkg/knuu/knuu.go

```diff
@@ -160,6 +164,19 @@ func (k *Knuu) handleTimeout(ctx context.Context) error {
 	return nil
 }
 
+func validateOptions(opts Options) error {
+	// When Minio is set, K8sClient must be set too
+	// to make sure that there is only one source of truth for the k8s client
+	if opts.MinioClient != nil && opts.K8sClient == nil {
+		return ErrK8sClientNotSet
+	}
+
+	if opts.TestScope == "" && opts.K8sClient == nil {
+		return ErrTestScopeNotSet
+	}
```

Comment:
This isn't actually a problem. Now that you've moved the validation logic out you can see that this is simply the base case. You should be able to not set these. 

The case you want to check is this:
```suggestion
	if opts.TestScope != "" && opts.K8sClient != nil && opts.TestScope != opts.K8sClient.Namespace() {
		return ErrTestScopeMistMatch
	}
```

Reply:
well it was like this before, but in one of the comments you told me to move this default test scope to the e2e pkg: 
https://github.com/celestiaorg/knuu/pull/434/files/41fff2e769f1e40f1a336d265a1579265647fc47#diff-04a1accd1c12dafffe8d3ecc43a623cd0bf22d03d3561f7e04e296e64962048eR8-R11
that's the reason for receiving it from the user. I am more fan of making it easier for the users, so will put it back.